include/roboteam_utils/Print.h

```diff
@@ -0,0 +1,48 @@
+#ifndef RTT_ROBOTEAM_AI_SRC_UTILITIES_PRINT_H_
+#define RTT_ROBOTEAM_AI_SRC_UTILITIES_PRINT_H_
+
+#include <iostream>
+#include <iomanip>
+
+class Printer {
+ public:
+  static void print(const std::string& color, const std::string& type, const std::string& file, const std::string& func, int line, const std::string& txt) {
+      auto fileTxt =  file + ":" + std::to_string(line);
+
+      std::cout << color << std::setw(10) << std::left;
+      std::cout << type << std::setw(32) << std::left;
+      std::cout << fileTxt << std::setw(32) << std::left;
+      std::cout << txt;
+      std::cout << "\033[0m" << std::endl;
```

Comment:
```suggestion
      std::cout << color << std::setw(10) << std::left
                     << type << std::setw(32) << std::left
                     << file << ':' << line << std::setw(32) << std::left
                     << txt;
                     << "\033[0m" << std::endl;
```

the colors arent' exactly portable but we only use linux anyway so whatever